main.py

```diff
@@ -67,11 +69,11 @@ def register():
     return render_template('register.html')
 
 
-@app.route('/cabinet')
+@app.route('/cabinet', methods=['GET', 'POST'])
 @login_required
 def cabinet():
     if request.method == "POST":
-        return wl_search(db)
```

Comment:
Don't understand where the search function goes. Does id search work?

Reply:
wl_search is located in search.py . It gets id from request.form["search"], I hope...)